packages/veritone-react-common/src/components/OCREngineOutputView/styles.scss

```diff
@@ -0,0 +1,24 @@
+.ocrOutputView {
+  display: flex;
+  flex-direction: column;
+}
+
+.ocrViewHeader {
```

Comment:
I think you can remove everything apart from `display: flex` and `.headerTitle` here